tests/Melodiia/Doctrine/DoctrineDataStoreTest.php

```diff
@@ -11,57 +11,66 @@
 use Pagerfanta\Pagerfanta;
 use PHPUnit\Framework\TestCase;
 use Prophecy\Argument;
+use Prophecy\Prophet;
 use SwagIndustries\Melodiia\Crud\FilterCollection;
 use SwagIndustries\Melodiia\Crud\Persistence\DataStoreInterface;
 use SwagIndustries\Melodiia\Doctrine\DoctrineDataStore;
 
 class DoctrineDataStoreTest extends TestCase
 {
-    public function testItIsAMelodiiaDataStore()
+    /** @var Prophet */
+    private $prophet;
+
+    public function setUp(): void
+    {
+        $this->prophet = new Prophet();
+    }
```

Comment:
Can you please use this package instead ? https://packagist.org/packages/phpspec/prophecy-phpunit

Reply:
I don't know about Symfony 5, i will test it and complete your suggestions asap :+1: